src/main/java/core/basesyntax/strategy/transaction/TransactionHandler.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.strategy.transaction;
+
+public interface TransactionHandler {
+    int getTransaction(int oldQuantity, int transactionQuantity);
```

Comment:
name of method doesn't reflect what it's doing
```suggestion
    int getTransactionResult(int oldQuantity, int transactionQuantity);
```